src/c++/vdo/fake/linux/bvec.h

```diff
@@ -11,6 +11,7 @@
 #define __LINUX_BVEC_H
 
 #include <linux/highmem.h>
+#include <linux/min_heap.h>
 #include <linux/minmax.h>
```

Comment:
Do you actually need min_heap.h here? I thought we worked around that.

Reply:
WARN_ONCE is defined in min_heap.h so